output/tps_style.xml

```diff
@@ -0,0 +1,146 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
Do we need this style file?  Or the template version of the style file that is populated with the aspects that depend on the input data, like the number of significant and insignificant peptides?

Reply:
This style file is used by the workflow by default. This is something I initially ignored but we can look to resolve this for the pull request. I could add the functionality where the style file is being generated with the input data rather than using the default. Let me know what you think about this.